pallets/parachain-system/src/relay_state_snapshot.rs

```diff
@@ -24,6 +24,17 @@ use sp_state_machine::{Backend, TrieBackend, TrieBackendBuilder};
 use sp_std::vec::Vec;
 use sp_trie::{HashDBT, MemoryDB, StorageProof, EMPTY_PREFIX};
 
+/// The capacity of the upward message queue of a parachain on the relay chain.
+// The field order should stay the same as the data can be found in the proof to ensure both are
+// have the same encoded representation.
+#[derive(Clone, Encode, Decode, TypeInfo, Default)]
+pub struct RelayDispachQueueSize {
+	/// The number of additional messages that can be enqueued.
+	pub remaining_count: u64,
+	/// The total size of additional messages that can be enqueued.
+	pub remaining_size: u64,
```

Comment:
I think the relay still uses `u32` for those (although should be changed to `u64` eventually), so how can this work?

Reply:
I can check again, but it is u64 afair. While I don't get why this is u64 as we will never have soo many remaining messages.

Reply:
You were right, it is `u32`. Fixed!